README.md

```diff
@@ -1,2 +1,340 @@
+
+In the easy start section we used default settings, but you will want to set your own values for things like signing key.  You'll set those in a .env file that is passed into the docker-run command:
+
+```
+docker run --env-file .env -dp 3000:4007 digitalcredentials/issuer 
+```
+
+### Environment Variables
+
+There is a sample .env file provided called .env.example to help you get started with your own .env file. The supported fields:
+
+| Key | Description | Default | Required |
+| --- | --- | --- | --- |
+| `PORT` | http port on which to run the express app | 4007 | no |
+| `ENABLE_HTTPS_FOR_DEV` | runs the dev server over https - ONLY FOR DEV - typically to allow CORS calls from a browser | false | no |
+| `ENABLE_STATUS_ALLOCATION` | enables the allocation of a StatusList2021 position.  If true, then the CRED_STATUS_* values must also be set | false | no |
+| `CRED_STATUS_OWNER` | name of the owner account (personal or organization) in the source control service that will host the credential status resources | no | yes if ENABLE_STATUS_ALLOCATION is true |
+| `CRED_STATUS_REPO_NAME` | name of the credential status repository | no | yes if ENABLE_STATUS_ALLOCATION is true |
+| `CRED_STATUS_META_REPO_NAME` | name of the credential status metadata repository | no | yes if ENABLE_STATUS_ALLOCATION is true |
+| `CRED_STATUS_ACCESS_TOKEN` | Github access token for the credential status repositories | no | yes if ENABLE_STATUS_ALLOCATION is true |
+| `CRED_STATUS_DID_SEED` | seed used to deterministically generate DID | no | yes if ENABLE_STATUS_ALLOCATION is true |
+| `TENANT_TOKEN_{TENANT_NAME}` | see [tenants](#tenants) section for instructions | no | at least one must be set |
+| `TENANT_SEED_{TENANT_NAME}` | see [tenants](#tenants) section for instructions | no | no |
+
+
+### Tenants
+
+You might want to allow more than one signing key/DID to be used with the issuer. For example, you might want to sign university/college degree diplomas with a DID that is only used by the registrar, but then also allow certificates for individual courses to be signed by by different DIDS that are owned by the faculty or department that teaches the course.
+
+We're calling these differents signing authorities 'tenants'.  You can set up as many tenants as you like by including a `TENANT_SEED_{TENANT_NAME}` environment variable for every 'tenant', and optionally a `TENANT_TOKEN_{TENANT_NAME}` environment variable for whichever tenants you want to secure with a simple bearer auth token.
+
+So, if you wanted to set up two tenants, one for degrees and one for completion of the Econ101 course, and you wanted to secure the degrees tenant but not the Econ101, then you could create the tenants by setting the following in the .env file:
+
+```
+TENANT_SEED_DEGREES=z1AoLPRWHSKasPH1unbY1A6ZFF2Pdzzp7D2CkpK6YYYdKTN
```

Comment:
We should add the DIDs generated from these example seeds, as well as the default seeds in `.env`, to the Community registry, something like "TEST ONLY: DCC Signing Service Default DID"

Reply:
Agree, but maybe we should talk to everyone else (Kerri, Gillian, Brandon, Philipp, Kayode) about it in case there is some reason not to?